docs/user/alerting/create-and-manage-rules.asciidoc

```diff
@@ -0,0 +1,173 @@
+[role="xpack"]
+[[create-and-manage-rules]]
+== Create and manage rules
+
+The *Rules* UI provides a cross-app view of alerting. Different {kib} apps like {observability-guide}/create-alerts.html[*Observability*], {security-guide}/prebuilt-rules.html[*Security*], <<geo-alerting, *Maps*>> and <<xpack-ml, *Machine Learning*>> can offer their own rules. The *Rules* UI provides a central place to:
+
+* <<create-edit-rules, Create and edit>> rules
+* <<controlling-rules, Manage rules>> including enabling/disabling, muting/unmuting, and deleting
+* Drill-down to <<rule-details, rule details>>
+
+[role="screenshot"]
+image:images/rules-and-connectors-ui.png[Example rule listing in the Rules and Connectors UI]
+
+For more information on alerting concepts and the types of rules and connectors available, see <<alerting-getting-started>>.
+
+[float]
+=== Required permissions
+
+Access to rules is granted based on your privileges to alerting-enabled features. See <<alerting-security, Alerting Security>> for more information.
+
+[float]
+[[create-edit-rules]]
+=== Create and edit rules
+
+Many rules must be created within the context of a {kib} app like <<metrics-app, Metrics>>, <<xpack-apm, APM>>, or <<uptime-app, Uptime>>, but others are generic. Generic rule types can be created in the *Rules* management UI by clicking the *Create* button. This will launch a flyout that guides you through selecting a rule type and configuring its properties. Refer to <<stack-rules, Stack rules>> for details on what types of rules are available and how to configure them.
+
+After a rule is created, you can re-open the flyout and change a rule's properties by clicking the *Edit* button shown on each row of the rule listing.
+
+When you create a rule, you must define the rule details, conditions, and actions.
```

Comment:
I think you should remove lines 29 through 36.  

If you do, I'd then change this sentence:

"...selecting a rule type and configuring its properties."

to

"selecting a rule type and configuring its conditions and action type."

Reply:
Done in this commit: https://github.com/elastic/kibana/pull/101498/commits/2c93360d845faf0643053a6b5e532fa5b2f614b3